web/src/condense.ts

```diff
@@ -290,10 +317,21 @@ export function initialize() {
     $("#message_feed_container").on("click", ".message_condenser", function (e) {
         const $row = $(this).closest(".message_row");
         const id = rows.id($row);
+        if (typeof id !== "number") {
+            return;
+        }
+
         // Focus on the condensed message.
-        assert(message_lists.current !== undefined);
-        message_lists.current.select_id(id);
-        message_lists.current.get(id).condensed = true;
+        if (message_lists.current !== undefined) {
+            message_lists.current.select_id(id);
+            const message = message_lists.current.get(id);
+            if (message !== undefined) {
```

Comment:
> There is no need to check if the message is undefined or not here is why.
we are checking if we are doing assertation above (line 328) to see if current is undefined or not.
which means, if message_lists exists then there exists a valid id 

here to